data/mapcss-mapping.csv

```diff
@@ -365,8 +365,8 @@ sport|9pin;364;
 sport|10pin;365;
 piste:type|downhill|novice;[piste:type=downhill][piste:difficulty=novice];;name;int_name;366;
 railway|narrow_gauge|bridge;[railway=narrow_gauge][bridge?];;name;int_name;367;
-deprecated|deprecated;368;x
-deprecated|deprecated;369;x
+sport|padel;368;
+leisure|pitch|padel;[leisure=pitch][sport=padel];x;name;int_name;369;leisure|pitch
```

Comment:
- keep only sport|padel here and below for all other sport types
- move them together somewhere after our current sports block. Less numbers are reserved for possible more frequent types.

Reply:
Done